docs/developer/first_delivery.md

```diff
@@ -0,0 +1,241 @@
+import { filesChangedSince, goal, GoalInvocation, PushListenerInvocation, pushTest } from "@atomist/sdm";
+
+const modifiesReadme = pushTest(
+  "modifiesReadme",
+  async (pushListenerInvocation: PushListenerInvocation) => {
+    const changedFiles = await filesChangedSince(pushListenerInvocation.project, pushListenerInvocation.push);
+    return changedFiles.includes("README.md");
+  });
+```
+
+## Attaching to your SDM
+
+We've set up a goal and defined when we want it to be invoked; it's time to associate it with our SDM!
+
+The _lib/machine/machine.ts_ file is used to control the behavior of the SDM. Here, we can define which events we want to listen to and which goals to run for those events.
+
+Let's start by importing the two functions that we defined earlier:
+
+```typescript
+import {
+    messageGoal, modifiesReadme,
+} from "./message";
+```
+
+Next, within the `machine` method, enter the following line:
+
+```typescript
+sdm.withPushRules(
+    whenPushSatisfies(modifiesReadme).setGoals(messageGoal));
+```
+
+The method names provide clarity on the configuration:
+
+* we configure the SDM to listen to push events (`withPushRules`)
+* we test that the push in question passes our previous filter (`whenPushSatisfies`)
+* if the push does pass the filter, we set the goals we want the SDM to achieve (`setGoals`)
+
+That's all there is to it!
+
+## Associating the SDM with your repository
+
+Now that our SDM is set up the behave just as we want it to, it's time to test everything and make sure that our code is correct. Before doing that, though, it's important to go over the differences between [Local Mode](/developer/local/) and [Team Mode](/developer/team/). Local Mode runs in the privacy of your own machine. The SDM doesn't communicate with the online Atomist service, and it behaves just like a local instance of the SDM would. Crucially, since we're operating locally, push events are simulated as coming from _commits_.
+
+To associate the local `atomist-seeds/express-es6-rest-api` repository with the SDM running Local Mode, you'll need to navigate to the folder and enter:
+
+```
+atomist enable local
+```
+
+This instructs the SDM to watch for events that this repository triggers.
+
+## Testing in Local Mode
+
+The next step is to test that a README altering commit prints a message to the terminal!
+
+Open a new terminal window and type:
+
+```
+atomist start --local
+```
+
+This starts Atomist in Local Mode. A whole slew of logging information will be printed, but you'll know the SDM is ready when the following line is shown:
+
+```
+Atomist automation client startup completed
+```
+
+Now, in another terminal, enter:
+
+```
+atomist feed
+```
+
+This command acts as a sort of `tail` for messages that the SDM sends. Any SDM activity that has to deal with events or the execution of goals shows up here.
+
+In your `atomist-seeds/express-es6-rest-api` folder, make a commit that modifies the README. You should see your message printed:
```

Comment:
Because I was told to fork then `atomist clone`, this isn't in `atomist-seeds` for me. Prolly just shorten to `express-es6-rest-api`.